### PR TITLE
Don't be shallow

### PR DESCRIPTION
**Pull request by eile**

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -11,7 +11,6 @@ if(NOT EXISTS ${CMAKE_SOURCE_DIR}/CMake/common/Common.cmake)
   message(FATAL_ERROR "CMake/common missing, run: git submodule update --init")
 endif()
 
-
 include(GitExternal)
```
Comment (hernando) [inline CMakeLists.txt]: remove

Comment (eile) [inline CMakeLists.txt]: What?

Comment (hernando) [inline CMakeLists.txt]: include(GitExternal) is not needed

Comment (rdumusc) [inline CMakeLists.txt]: remove include(GitExternal)

Comment (eile) [inline CMakeLists.txt]: Ah, I thought the newline (blush)

**Comment by hernando** [comment]
Retest this please

**Pull request merged**